.obs/seedimage/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+# Define the names/tags of the container
+#!BuildTag: rancher/seedimage-builder/5.3:latest
+#!BuildTag: rancher/seedimage-builder/5.3:%OPERATOR_VERSION%
+#!BuildTag: rancher/seedimage-builder/5.3:%OPERATOR_VERSION%-%RELEASE%
+#
+
+FROM suse/sle15:15.4 as BASE
+
+RUN mkdir /installroot
+RUN  mkdir -p /installroot/etc/products.d && \
+    cp /etc/products.d/baseproduct /installroot/etc/products.d/ && \
+    zypper --gpg-auto-import-keys --installroot /installroot in -y --no-recommends xorriso httpfy curl coreutils ca-certificates && \
+    cp -a /var/lib/ca-certificates/openssl/* /installroot/var/lib/ca-certificates/openssl/
```

Comment:
I don't think this is needs if you include the `ca-certificates-mozilla` package.